core/renderers/common/i_path_object.js

```diff
@@ -90,14 +90,6 @@ Blockly.blockRendering.IPathObject.prototype.setMarkerSvg;
  */
 Blockly.blockRendering.IPathObject.prototype.updateHighlighted;
 
-/**
- * Set whether the block shows a disable pattern or not.
- * @param {boolean} disabled True if disabled.
- * @param {boolean} isShadow True if the block is a shadow block.
- * @package
```

Comment:
I think this should still be here, just marked protected.

Reply:
The reason I removed this is because I dont think it is a must that each renderer define these methods. For example, a renderer might choose to override the ``applyColours`` method and use it's own internal set of methods to determine how colours are applied, or not use helpers at all. It's not important for the public interface to require these "helper" methods to be present / implemented.

Reply:
I'm torn, because the base `applyColours` is calling them.  But I see your point for a renderer that overrides `applyColours` entirely.  So lgtm, approval.